src/github.org

```diff
@@ -0,0 +1,99 @@
+- Git tracks changes made to files in your project, and handles merging
+  these changes
+- Working on different parts of code and then combining them creates unique challenges that
+  cannot be handled by file sharing systems like DropBox and Google Drive
+
+* What is GitHub?
+- GitHub is a publically hosted git server
+- It hosts our projects as "repositories" on the internet
+- This saves us the hassle of hosting a server for fellow RoboJackets to
+  obtain copies of the code
+- As you matriculate through Tech, you'll be building an online portfolio of
+  your work on GitHub.
+
+* How do you find things on GitHub?
+- Repositories are stored at https://github.com/<ORG-OR-USER-NAME>/<REPO-NAME>
+- So all of our projects can be found at https://github.com/RoboJackets
+- Your personal projects will exist at https://github.com/<YOUR-GH-USERNAME>
+
+** Create a GitHub account
+- [[https://github.com/join%0A][https://github.com/join]]
+- Please include /at least/ your real first name
+# Emphasize this requirement is for display name on GitHub, not username
+
+[[file:https://i.imgur.com/0cdXQXW.png]]
+
+* Fork and star our repo
+- Navigate to github.com/robojackets/robocup-software
+    - Or the repo of whatever RoboJackets team you are on
+
+[[file:https://i.imgur.com/kYzz2oh.png]]
+
+* What is a fork?
+- Problem: How do I try something out on a project without affecting the project itself?
+- Solution: Forks!
+- A fork is a copy of a repository that exists on your personal account
+- Changes made on a fork will not affect the original repository, and vice versa
+- Your fork is present on https://github.com/<YOUR-GH-USERNAME>/robocup-software
+
+* Install Git on your computer
+- Windows: https://git-scm.com/download/win
+- Mac OS: https://git-scm.com/download/mac
+- Linux: https://git-scm.com/download/linux
+
+* Git Config
+- These commands set up git to attribute your contributions to our code to you.
+
+  #+BEGIN_SRC shell
+    git config --global user.name "Your Name"
+    git config --global user.email "your@email.here"
+  #+END_SRC
+
+* Let's grab some code!
+- Go back to the webpage for our GitHub repository
+- Select the clone button, then copy the link displayed
+[[file:https://i.imgur.com/n3h8Tb5.png]]
+
+** Clone the repository
+- In your terminal navigate to your workspace
+
+#+BEGIN_SRC shell
+git clone <PASTE-CLIPBOARD-CONTENTS-HERE>
+#+END_SRC
+
+- Tada! Your own working copy of RoboJackets code!
+
+* Adding Remotes
+- These commands set you up to push to your fork by default, and still be able to receive updates.
+
+# Emphasize that the brackets are to be replaced
+#+BEGIN_SRC shell
+  cd ~/<REPOSITORY-FOLDER>
+  git remote add rj https://github.com/RoboJackets/robocup-software.git
+  git remote set-url origin https://github.com/<YOUR-GH-USERNAME>/robocup-software.git
+  git remote -v
+
+  # You can find the link in the last command under the 'clone or download'
+  # green button on your FORK
```

Comment:
There was a huge amount of people last year who couldn't find the green button. The web url should work too in this case, maybe we can just tell people to grab the URL from their browser?